README.md

```diff
@@ -16,20 +16,38 @@
 
 <!-- TODO nf-core: Add full-sized test dataset and amend the paragraph below if applicable -->
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/metatdenovo/results).
+On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/metatdenovo/results) e.g. the results for running the pipeline with --assembler will be in a folder called orf_caller and so on.
 
 ## Pipeline summary
 
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
-
+![nf-core/rnaseq metro map](docs/images/nf-core-metatdenovo_metro_map.png)
 1. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
 2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+3. Quality trimming and adapters removal for raw reads ( [`Trimm Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/))
+4. Merge pair-end for cleaned reads ( [`Seqtk`](https://github.com/lh3/seqtk))
+5. Choice of de novo assembly program route:
+    1. [`RNAspade`](https://cab.spbu.ru/software/rnaspades/) Suggested for Eukaryotes de novo assembly
+    2. [`Megahit`](https://github.com/voutcn/megahit) Suggested for Prokaryotes de novo assembly
+6. Orf caller step:
+    1. [`TransDecoder`](https://github.com/TransDecoder/TransDecoder) suggested for Eukaryotes
+    2. [`Prokka`](https://github.com/tseemann/prokka) suggested for Prokaryotes
+    3. [`Prodigal`](https://github.com/hyattpd/Prodigal) suggested for Prokaryotes
+7. quantification route:
+    1. generate index of assembly [`BBmap index`](https://sourceforge.net/projects/bbmap/) 
+    2. Mapping cleaned reads to the assembly for quantification [`BBmap`](https://sourceforge.net/projects/bbmap/) 
+    3. Get raw counts per orf samples [`Featurecounts`](http://subread.sourceforge.net) -> TSV table with collected featurecounts output 
```

Comment:
What does "per orf samples" mean?

Reply:
It should be more "per each gene present in the assembly" or something similar